core/src/main/java/org/jdbi/v3/core/statement/SqlStatements.java

```diff
@@ -222,11 +222,11 @@ public boolean isUnusedBindingAllowed() {
      * Defaults to false: unused bindings are not allowed.
      *
      * @see org.jdbi.v3.core.argument.Argument
-     * @param allowUnusedBindings the new setting
+     * @param unusedBindingAllowed the new setting
      * @return this
      */
-    public SqlStatements setUnusedBindingAllowed(boolean allowUnusedBindings) {
-        this.allowUnusedBindings = allowUnusedBindings;
+    public SqlStatements setUnusedBindingAllowed(boolean unusedBindingAllowed) {
+        this.allowUnusedBindings = unusedBindingAllowed;
```

Comment:
This is a good example of why I'm worried this kind of rule might be over-applied.  We're now permuting components of our terms just so that they are different and unique, but really they are the naming the same concept.

Reply:
Huh, I kinda wonder why this was renamed. It should pass through the `setter` exclusion.

Reply:
Ah, wait, of course. This parameter had to be changed because it didn't match the setter method name. The real problem here is that the setter doesn't match the field name, and this rule just forces the parameter to stick to the method instead of the field. I agree with your remark but it's applying to the symptom of a whole other problem here, not to the cause.